Formula/pymupdf.rb

```diff
@@ -0,0 +1,61 @@
+class Pymupdf < Formula
+  desc "Python bindings for the PDF toolkit and renderer MuPDF"
+  homepage "https://github.com/pymupdf/PyMuPDF"
+  url "https://files.pythonhosted.org/packages/29/e4/d1d88146ef0b3b97d785acc7aed22b9774ac6bcf137e98b48a9c9bbb7f35/PyMuPDF-1.20.1.tar.gz"
+  sha256 "305c1a64b8fb2fd465e27cc8bdcbf0f64224f0ec6d7763e3f5f2ca6783136649"
+  license "AGPL-3.0-only"
+
+  depends_on "freetype" => :build
+  depends_on "swig" => :build
+
+  depends_on "mupdf"
+  depends_on "python@3.10"
+
+  on_linux do
+    depends_on "mujs"
+    depends_on "openjpeg"
+    depends_on "gumbo-parser"
+    depends_on "harfbuzz"
+    depends_on "jbig2dec"
+  end
+
+  def install
+    if OS.linux?
+      ENV.append "CPATH", Formula["mupdf"].include/"mupdf", ":"
+      ENV.append "CPATH", Formula["freetype2"].include/"freetype2", ":"
+    end
+
+    # Makes setup skip build stage for mupdf
+    # https://github.com/pymupdf/PyMuPDF/blob/1.20.0/setup.py#L447
+    ENV["PYMUPDF_SETUP_MUPDF_BUILD"] = ""
+
+    system "python3", *Language::Python.setup_install_args(prefix),
+                      "--install-lib=#{prefix/Language::Python.site_packages("python3")}",
+                      "build"
```

Comment:
```suggestion
    system "python3", *Language::Python.setup_install_args(prefix), "build"
```
We don't need `install-lib` anymore.